packages/perseus-core/package.json

```diff
@@ -30,4 +31,4 @@
     },
     "peerDependencies": {},
     "keywords": []
-}
+}
```

Comment:
OCD-level 10 comment:  looks like all the package.json lost their newline at end of file. I'm not sure if that matters, but Github flags it so... 🤷‍♂️

Reply:
It doesn't make a difference. I just saved like usual - no idea why this means it doesn't get a newline.